web-marketplace/src/components/organisms/RegistryLayout/hooks/useHasPrefinanceProjects.ts

```diff
@@ -0,0 +1,32 @@
+'use client';
```

Comment:
could we actually prefetch in a server component?
this would prevent the UI flickering, the header label is currently briefly switching from "Projects" to "Projects ▼"